code/controllers/subsystem/mapping.dm

```diff
@@ -132,7 +132,7 @@ SUBSYSTEM_DEF(mapping)
 		var/msg = "RED ALERT! The following map files failed to load: [FailedZs[1]]"
```

Comment:
^ but it obviously did happen otherwise how would we know to fix the error message!